packages/base/Tabs/src/Tabs/Tabs.tsx

```diff
@@ -26,56 +23,83 @@ export interface Props
   orientation?: 'horizontal' | 'vertical'
 
   /** Determines additional display behavior of the tabs */
-  variant?: Extract<TabsProps['variant'], 'scrollable' | 'fullWidth'>
+  variant?: 'scrollable' | 'fullWidth'
 }
 
-const useStyles = makeStyles<Theme>(styles, {
-  name: 'Tabs',
-})
-
-export const TabsOrientationContext = React.createContext<
-  'horizontal' | 'vertical'
->('horizontal')
+export const TabsContext = React.createContext<{
+  orientation: 'horizontal' | 'vertical'
+  variant: 'scrollable' | 'fullWidth'
+}>({ orientation: 'horizontal', variant: 'scrollable' })
 
 // eslint-disable-next-line react/display-name
-export const Tabs = forwardRef<HTMLButtonElement, Props>(function Tabs(
+export const Tabs = forwardRef<HTMLDivElement, Props>(function Tabs(
   props,
   ref
 ) {
   const {
     children,
-    orientation,
+    orientation = 'horizontal',
     onChange,
     value,
     variant = 'scrollable',
+    className,
     ...rest
   } = props
-  const classes = useStyles(props)
-  const action = useTabAction()
+
+  const [contextValue] = useState({
+    orientation,
+    variant,
+  })
```

Comment:
Why `useState` here? We don't change the value anywhere. Perhaps you want to use it for memorizing the value?
If that is the case, changing `orientation` `variant` will not trigger a new `contextValue`, meaning the initial values for these props are stuck forever, maybe replace it with `useMemo`?
```suggestion
  const [contextValue] = useMemo(() => ({
    orientation,
    variant,
  }), [orientation, variant])
```

Reply:
good catch, thanks 🙇